share/wake/lib/core/list.wake

```diff
@@ -380,11 +395,13 @@ export def append (item: a) (list: List a): List a =
 export def splitAt (index: Integer) (listToDivide: List a): Pair (List a) (List a) =
     if index <= 0 then
         Pair Nil listToDivide
-    else match listToDivide
-        Nil  = Pair Nil Nil
-        h, t =
-            def Pair u v = splitAt (index-1) t
-            Pair (h, u) v
+    else
+        match listToDivide
```

Comment:
Do we want to give `else match` the same sort of exception as allows `else if`?  (Also, how does `require` play into that exception?)

Reply:
easy enough to do. Just hadn't considered it.

Not sure I see the difference between `def x = match` and `else match` though